recipes/rmlui/4.x/conanfile.py

```diff
@@ -0,0 +1,170 @@
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if not hasattr(self, "_cmake"):
+            self._cmake = CMake(self)
+            self._cmake.definitions["BUILD_LUA_BINDINGS"] = self.options.with_lua_bindings
+            self._cmake.definitions["BUILD_SAMPLES"] = False
+            self._cmake.definitions["CUSTOM_CONFIGURATION"] = True
+            self._cmake.definitions["CUSTOM_INCLUDE_DIRS"] = ";".join(self.deps_cpp_info["robin-hood-hashing"].include_paths)
+            self._cmake.definitions["DISABLE_RTTI_AND_EXCEPTIONS"] = not self.options.enable_rtti_and_exceptions
+            self._cmake.definitions["ENABLE_PRECOMPILED_HEADERS"] = True
+            self._cmake.definitions["ENABLE_TRACY_PROFILING"] = False
+            self._cmake.definitions["MATRIX_ROW_MAJOR"] = self.options.matrix_mode == "row_major"
+            self._cmake.definitions["NO_FONT_INTERFACE_DEFAULT"] = self.options.font_interface is None
+            self._cmake.definitions["NO_THIRDPARTY_CONTAINERS"] = not self.options.with_thirdparty_containers
+
+            self._cmake.configure()
+
+        return self._cmake
+
+    def _patch_sources(self):
+        # The *.cmake files that conan generates using cmake_find_package for CMake's find_package to consume use
+        # different variable naming than described in CMake's documentation, thus the need for most of the replacements.
+        # References:
+        #  * https://cmake.org/cmake/help/latest/module/FindFreetype.html
+        #  * https://cmake.org/cmake/help/latest/module/FindLua.html
+        replace_mapping = {
+            "FREETYPE_FOUND": "Freetype_FOUND",
+            "FREETYPE_INCLUDE_DIRS": "Freetype_INCLUDE_DIRS",
+            "FREETYPE_LINK_DIRS": "Freetype_LINK_DIRS",
+            "FREETYPE_LIBRARY": "Freetype_LIBRARIES",
+            "LUA_FOUND": "lua_FOUND",
+            "LUA_INCLUDE_DIR": "lua_INCLUDE_DIR",
+            "LUA_LIBRARIES": "lua_LIBRARIES",
+            # disables the built-in generation of package configuration files
+            "if(PkgHelpers_AVAILABLE)": "if(FALSE)"
+        }
+
+        cmakelists_path = os.path.join(
+            self._source_subfolder, "CMakeLists.txt")
+        for key, value in replace_mapping.items():
+            tools.replace_in_file(cmakelists_path, key, value)
+
+        if self.options.with_thirdparty_containers:
+            config_path = os.path.join(self._source_subfolder,
+                                       "Include", "RmlUi", "Config", "Config.h")
+            tools.replace_in_file(
+                config_path, "\"../Core/Containers/robin_hood.h\"", "<robin_hood.h>")
+
+    def build(self):
+        self._patch_sources()
+        self._configure_cmake().build()
+
+    def package(self):
+        self._configure_cmake().install()
+        self.copy("*LICENSE.txt", dst="licenses", src=self._source_subfolder, excludes=("Samples/*", "Tests/*"))
+
+    def package_info(self):
```

Comment:
The cmake find package is wrong

https://github.com/mikke89/RmlUi/blob/master/CMakeLists.txt#L909

Conan generate will have the wrong casing

Reply:
I personally use the library through Conan, so I didn't fiddle with settings related to CMake's find_package.